packages/versions/src/lib/checkFuelCoreVersionCompatibility.ts

```diff
@@ -4,6 +4,12 @@ import { majorEq, minorEq, patchEq } from './semver';
 export function checkFuelCoreVersionCompatibility(networkVersion: string) {
   const { FUEL_CORE: supportedVersion } = getBuiltinVersions();
 
+  if (networkVersion.match(/^[0-9]+\.[0-9]+\.[0-9]+$/) === null) {
+    console.warn(
```

Comment:
The `console` lint errors can be circumvented by the usage below.

It seems more elegant than inline-disabling the rule.

```ts
import { warn } from 'console';

warn(...)
```

Reply:
I've got a rollup error that doesn't allow me to do this:
```
fuels:build: RollupError: "warn" is not exported by "__vite-browser-external", imported by "../versions/dist/index.mjs".
```

I'll have to go the inline disabling route unfortunately.

Reply:
Not 100%, but you may need to polyfill `console` in browser which you can add [here](https://github.com/FuelLabs/fuels-ts/blob/master/vite.browser.config.mts#L15).